packages/editor/src/components/post-title/index.js

```diff
@@ -170,7 +169,7 @@ function PostTitle( _, forwardedRef ) {
 				( firstBlock.name === 'core/heading' ||
 					firstBlock.name === 'core/paragraph' )
 			) {
-				onUpdate( stripHTML( firstBlock.attributes.content ) );
+				onUpdate( firstBlock.attributes.content );
```

Comment:
```suggestion
				onUpdate( stripHTML( firstBlock.attributes.content ) );
```

I'm not sure we should revert this. https://github.com/WordPress/gutenberg/pull/35825 solved a legitimate problem with posts being inadvertently created with HTML in the title and the slug.